kpi/views/environment.py

```diff
@@ -38,6 +40,18 @@ class EnvironmentView(APIView):
             'OPERATIONAL_PURPOSE_CHOICES',
             lambda text: tuple((line.strip('\r'), line.strip('\r')) for line in text.split('\n')),
         ),
+        (
+            'MFA_I18N_HELP_TEXTS',
+            lambda i18n_texts: {
+                lang: markdown(text)
+                for lang, text in json.loads(
+                    i18n_texts.replace(
+                        '##support email##', constance.config.SUPPORT_EMAIL
```

Comment:
unsure we should substitute `##support email##` here since the superuser has full control over the entire string. administrators may not want to use the same address for general support as for MFA reset issues

Reply:
boneheaded comment on my part. if they want a custom email address, they can simply use that and omit `##support email##` from their string.